environment.yml

```
@@ -22,4 +21,4 @@ dependencies:
   - sphinx-jsonschema==1.15
   - sphinx <7.1.2
   - pip:
-    - autodoc_pydantic<2.0.0
+    - git+https://github.com/openforcefield/openff-models@pydantic-2-redo
```

Could you also add this before I cut the release, just as an extra layer of safety?
```suggestion
    - git+https://github.com/openforcefield/openff-models@pydantic-2-redo
    - git+https://github.com/openforcefield/openff-units@9755af2978374938f489385c784b2dd2a056e2bc
```

---

Thanks!

---

Odd versioneer does not like that

---

Looks like py3.9 being silly.

---

We don't need py3.9 -- removing it 🔪